packages/cli/src/generator/templates.js

```diff
@@ -126,29 +116,23 @@ function installTemplateDependencies(templatePath, yarnVersion) {
 
   let dependencies;
   try {
-    dependencies = JSON.parse(fs.readFileSync(dependenciesJsonPath));
+    dependencies = JSON.parse(fs.readFileSync(dependenciesJsonPath).toString());
```

Comment:
If it's json you can require it

Reply:
Literally changed that because of flow errors. Didn't even thing about it :P

Reply:
I get that, but you know, while we're at it ;)